engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByOperatorFactory.java

```diff
@@ -519,6 +521,21 @@ public Void visit(@NotNull final RollingCountSpec spec) {
             return null;
         }
 
+        @Override
+        public Void visit(@NotNull final RollingFormulaSpec spec) {
+            final boolean isTimeBased = spec.revWindowScale().isTimeBased();
+            final String timestampCol = spec.revWindowScale().timestampCol();
```

Comment:
Probably something that should have been brought up in earlier review... but, do we expect `WindowScale#isTimeBased` and `WindowScale#timestampCol` to be the same for rev and fwd? If so, I'm assuming we should add that check to `RollingOpSpec` and potentially elevate isTimeBased / timestampCol helpers to RollingOpSpec? (This might effect construction and use of @Default)

Reply:
We don't do this currently because we don't require the user to supply a fwd window scale.  A default is provided of `WindowScale.ofTicks(0)` and we use only the `timestampCol()` from rev window scale (which is always required).

If we enforce this check, we need also to rewrite all `Rolling` op calls that skip the `fwd` parameter to construct a new `WindowScale` object with the matching timestamp column name.  This might be a good thing, but it isn't a trivial change.